src/link/rgblink.1

```diff
@@ -70,9 +77,11 @@ to fix these so that the program will actually run in a Game Boy:
 .D1 $ rgbfix -v bar.gb
 .Sh SEE ALSO
 .Xr rgbasm 1 ,
+.Xr rgblink 5 ,
 .Xr rgbfix 1 ,
 .Xr rgbds 7
 .Sh HISTORY
 .Nm
 was originally written by Carsten S\(/orensen as part of the ASMotor package,
-and was later packaged in RGBDS by Justin Lloyd.
+and was later packaged in RGBDS by Justin Lloyd. It is now maintained by a
+number of contributors on GitHub.
```

Comment:
So it does. My bad.

    .Lk https://github.com/rednex/rgbds GitHub .

Should work better. Will render as “GitHub: https://github.com/rednex/rgbds.” in the terminal, and “[GitHub](https://github.com/rednex/rgbds).” in HTML output.